File: PRAK205-2210817220011-MeldianaHandayani.c
#include <stdio.h>
#include <math.h>
 int main (){
    int A,B,Alas,Keliling,Luas;
    scanf("%d", &A);
    scanf("%d", &B);
    Alas= sqrt(pow(B,2)-pow(A,2));
    Keliling= Alas+A+B;
    Luas= 0.5* Alas*A;
    printf("Alas = %d cm\n", Alas);
    printf("Tinggi = %d cm\n", A);
    printf("Keliling = %d cm\n", Keliling);
    printf("Luas = %d cm^2\n", Luas);
 }

File: PRAK205-2210817220011-MeldianaHandayani.py
import math
A= int(input())
B=int(input())
Alas=round(math.sqrt(pow(B,2)-pow(A,2)))
Keliling= Alas+A+B
Luas= int(0.5* Alas*A)
print("Alas = {} cm".format (Alas))
print("Tinggi = {} cm".format(A))
print("Keliling = {} cm".format(Keliling))
print("Luas = {} cm^2".format (Luas))
